the check rule will trigger a weird dependency issue in automake,
it's possible that update-mime-database are invoked simultaneously in
two different threads at the same time.

As a workaround, revert it to the logic in 0.80

Upstream-Status: Inappropriate [workaround]

Signed-off-by: qing He <qing.he@intel.com>

diff --git a/Makefile.am b/Makefile.am
index 1a887bc..9df581e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -72,7 +72,7 @@ uninstall-hook:
 create-pot:
 	$(MAKE) -C po shared-mime-info.pot
 
-check: freedesktop.org.xml update-mime-database check-translations
+check: freedesktop.org.xml check-translations
 	if test -n $(XMLLINT) ; then \
 		if test -e $(top_builddir)/freedesktop.org.xml; then \
 		xmllint --noout --valid $(top_builddir)/freedesktop.org.xml;	\
@@ -80,9 +80,6 @@ check: freedesktop.org.xml update-mime-database check-translations
 		xmllint --noout --valid $(top_srcdir)/freedesktop.org.xml;	\
 		fi \
 	fi
-	if test -d .git/ && test -x $(top_builddir)/update-mime-database && test -d $(top_srcdir)/mime-db-tests ; then \
-		$(top_builddir)/update-mime-database mime-db-tests/ ; \
-	fi
 	if test -d .git/ && test -x ../xdgmime/src/test-mime-data && test -x $(top_builddir)/test-tree-magic ; then \
 		mkdir -p $(top_builddir)/temp-mime-dir/mime/packages ; \
 		cp -a $(top_builddir)/freedesktop.org.xml $(top_builddir)/temp-mime-dir/mime/packages/ ; \
